docs: antd components revision (#4642)
refinedev/refine: documentation/docs/api-reference/antd/components/breadcrumb.md

:::info Legacy Behavior (Dashboard Page)

In the earlier versions of **refine**, `<Refine>` component had accepted `DashboardPage`, which could be used to add an index page to your app. With the changes in the `routerProvider` API of **refine** however, `DashboardPage` is deprecated. You can now define an index route yourself manually by your router package.

> In earlier versions, the home icon in the `Breadcrumb` was created by the `DashboardPage`, now it is rendered if you define a an action route as `/` in any one of your resources. It will be rendered with the home icon regardless of the current route. You can also hide the home icon by setting `showHome` to `false`.

Author: Wouldn't this be better?
```suggestion
In earlier versions, the home icon in the `Breadcrumb` was created by the `DashboardPage`, now it is rendered anytime you define an action route as `/` in any one of your resources. It will be rendered with the home icon regardless of the current route. You can also hide the home icon by setting `showHome` to `false`.
```
